README.md

```diff
@@ -0,0 +1,26 @@
+This is a starter template for [Ionic](http://ionicframework.com/docs/) projects.
+
+## How to use this template
+
+*This template does not work on its own*. The shared files for each starter are found in the [ionic2-app-base repo](https://github.com/ionic-team/ionic2-app-base).
+
+To use this template, either create a new ionic project using the ionic node.js utility, or copy the files from this repository into the [Starter App Base](https://github.com/ionic-team/ionic2-app-base).

```

Comment:
I am confused... so I should clone the ionic2-app-base repo and overwrite the files from here to there?